solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -181,17 +191,28 @@ contract WalletRegistry is Ownable {
 
     /// @notice Requests a new wallet creation.
     /// @dev Can be called only by the owner of wallets.
-    ///      It starts a new DKG process.
+    ///      It locks the DKG and request a new random relay entry. It expects
+    ///      that the DKG process will be started once a new random relay entry
+    ///      gets generated.
     function requestNewWallet() external {
         require(msg.sender == walletOwner, "Caller is not the Wallet Owner");
```

Comment:
Non-blocking this PR but why not making it a modifier?

Reply:
There's just one place where we use it. If we use it in more places later we should add a modifier.